packages/discord/tests/bot.test.ts

```
@@ -921,6 +923,393 @@ Creating measurements:
+							query: 'nav=stats',
+						},
+					},
+				},
+				'tok3n',
+			);
+
+			expect(getMeasurementMock).toHaveBeenCalledWith('m345ur3m3nt');
+
+			const expectedText
+				= expectedResponse.results[0].result.rawOutput.trim().slice(0, 1000)
+				+ '\n... (truncated)';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)